scripts/install-sysdig.in

```diff
@@ -144,8 +144,11 @@ if [ -f /etc/debian_version ]; then
 	esac
 
 elif [ -f /etc/system-release-cpe ]; then
-	DISTRO=$(cat /etc/system-release-cpe | cut -d':' -f3)
-	VERSION=$(cat /etc/system-release-cpe | cut -d':' -f5 | cut -d'.' -f1 | sed 's/[^0-9]*//g')
+	IFS=':' read -ra RELEASE <<< $(</etc/system-release-cpe)
```

Comment:
This seems a more convoluted way of saying ```read -ra RELEASE < /etc/system-release-cpe```.  The here-string form seems to trim some white space, though.  Care to walk me through the reasoning?

Thanks